frontend/public/module/k8s/types.ts

```diff
@@ -1113,12 +1113,33 @@ export type NetworkPolicyPort = {
 export type ConsolePluginKind = K8sResourceCommon & {
   spec: {
     displayName: string;
+    backend: {
+      service: {
+        basePath: string;
+        name: string;
+        namespace: string;
+        port: number;
+      };
+      type: 'Service';
+    };
+    i18n: {
+      loadType: 'Preload' | 'Lazy' | '';
```

Comment:
`''` empty string is not valid `loadType` value

Reply:
// LoadType is an enumeration of i18n loading types
// +kubebuilder:validation:Enum:=Preload;Lazy